### PR TITLE
add shift+tab

### PR DESCRIPTION
**Pull request by fleytman**
https://github.com/magmax/python-readchar/issues/80

### CODE REVIEW

File: readchar/posix_key.py [hunk 1]
```diff
@@ -6,6 +6,7 @@
 SPACE = "\x20"
 ESC = "\x1b"
 TAB = "\x09"
+SHIFT_TAB = "\x1b\x5b\x5a"
```
Comment (Cube707) [inline readchar/posix_key.py:9]: please move this line down and start a new block for `SHIFT` combinations

add here:

https://github.com/Cube707/python-readchar/blob/6fa6d7d4c823600a80f08dd1caf940bba43d66eb/readchar/posix_key.py#L65-L67

like this:
```suggestion
# SHIFT
SHIFT_TAB = "\x1b\x5b\x5a"
```

**Comment by Cube707** [comment]
and please remove my `switch links` commit. thats just a temporary experiment for my fork and is not suposed to propegate into the master repo

**Comment by Cube707** [comment]
please run:

```bash
git fetch --all
git checkout patch-1
git cherry-pick 40e9727c7c85c30e6411215eb8c596a866b06922
git push -f
```

**Comment by Cube707** [comment]
the username used in the commit doesn't match your github profile, so they are not linked.

Do you care? If not I will merge

**Comment by fleytman** [comment]
Thanks, I fix it

**Pull request merged**